ompi/mpi/fortran/use-mpi-f08/Makefile.am

```diff
@@ -110,37 +122,18 @@ profile/psizeof_f08.f90:
 	    --complex4=$(OMPI_HAVE_FORTRAN_COMPLEX4) \
 	    --complex32=$(OMPI_HAVE_FORTRAN_COMPLEX32)
 
-CLEANFILES += sizeof_f08.h sizeof_f08.f90 profile/psizeof_f08.f90
+CLEANFILES += sizeof_f08.h sizeof_f08.f90 psizeof_f08.f90
 
 mpi_api_files = \
         abort_f08.F90 \
-        accumulate_f08.F90 \
```

Comment:
I didn't see where all these deleted files got re-added...?  Maybe I missed it...?